### ISSUE TITLE
The beta version has not been uploaded since June 15th

### ISSUE THREAD

**Issue by Discontinuum** [Dugy/Legend_of_the_Invincibles#331]

**Comment by Dugy** [comment]
If I recall correctly, it doesn't upload if no changes are made.

The fact that it _was_ uploaded on June 15th suggests that I managed to set the password right...?

**Comment by Discontinuum** [comment]
No. June 15th is the last day when old Travis CI made an upload. A new Github Actions script has been failing with the incorrect password every time so far.

**Comment by Dugy** [comment]
Aha. In that case, I need to reset the password.

**Comment by Discontinuum** [comment]
Hm, the beta is still not working.
I would like to make another proposal then, maybe it's possible to make a new stable release? For 1.14. 1.14 is soon to become outdated (approx. in November), but I would like it to have the best available LotI, with my item fixes. The stable hasn't been updates for a year. Is it possible to make a new stable release now?

**Comment by Dugy** [comment]
I have asked a forum admin to reset the password and nothing happened. It was a month ago. I expected him to reply, but I received exactly zero responses, I didn't want to spam the others, so I waited until I forgot about it. Sorry.

I can make a stable release, I remember the password for that one.

Because I don't want to have others download updates too often, do you have any little things you'd like to get into the stable release before I upload it?

**Comment by Discontinuum** [comment]
Hm, indeed there are several things I would like to handle before releasing. I will try to make them before the end of this week, so in the end of week the release will be possible

**Comment by Dugy** [comment]
Okay.

**Comment by Discontinuum** [comment]
As I've tested so far, everything was alright, so from my side all changes I would like to see are there, the road to release is open. The decision about exact timing is yours though

Also, as the add-on server for upcoming 1.16 has opened, you can set up a beta there. As you have no beta on 1.16 server so far, you'll be able to set up whatever password you like

**Comment by Discontinuum** [comment]
Meh, yeah, unfortunately some problems uncovered, so we can't to release. Concerning #345 , it depends on the way you want to fix it.
